File: ExamPatient/ExamPatient/Patient.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ExamPatient
{
    [Serializable]
    public class Patient
    {
        public string Name;
        public int Age;
        public int RoomNumber;

        public Patient()
        {

        }

        public Patient(string name, int age, int roomNumber)
        {
            Name = name;
            Age = age;
            RoomNumber = roomNumber;
        }
    }
}

File: ExamTest/app/src/main/java/com/example/examtest/Patient.java
package com.example.examtest;

public class Patient {
    int id;
    String name;
    int age;

    public Patient(String name, int age) {
        this.name = name;
        this.age = age;
    }
    public Patient(int id, String name, int age) {
        this.id = id;
        this.name = name;
        this.age = age;
    }
}
